CQSDIContainer/CQSDIContainer/Commands/ZDoNothingAndDoSomethingCommandHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CQSDIContainer.Attributes;
+using IQ.Platform.Framework.Common.CQS;
+
+namespace CQSDIContainer.Commands
+{
+	[LogExecutionTimeToConsole]
```

Comment:
These attributes decorate the command handler during instantiation by the factory.  There will be some decorators applied automatically (like wrapping all commands within a transaction scope), and some of those could be opt out.  All that logic is contained within the factory.  Your handler is only responsible for executing the actual work you wish to perform!